app/web/features/messages/requests/HostRequestView.tsx

```diff
@@ -158,17 +157,27 @@ export default function HostRequestView({
 
   const userSummarySection = (
     <>
-      <UserSummary user={otherUser} smallAvatar={isMobile}>
+      <UserSummary
+        className={classes.userSummary}
+        user={otherUser}
+        smallAvatar={isMobile}
+      >
         {hostRequest && (
           <div className={classes.requestedDatesWrapper}>
             <Typography
               component="p"
               variant="h3"
               className={classes.requestedDates}
             >
-              {`${dayjs(hostRequest.fromDate).format("LL")} - ${dayjs(
-                hostRequest.toDate
-              ).format("LL")}`}
+              {`${
+                isMobile
+                  ? dayjs(hostRequest.fromDate).format("ll")
+                  : dayjs(hostRequest.fromDate).format("LL")
+              } - ${
+                isMobile
+                  ? dayjs(hostRequest.fromDate).format("ll")
+                  : dayjs(hostRequest.toDate).format("LL")
+              }`}
```

Comment:
Makes the shortened form of the month name on mobile as it was wrapping to a second line otherwise.